docs/api/spec/query-api.yaml

```diff
@@ -0,0 +1,208 @@
+openapi: 3.0.0
+info:
+  title: Treetracker Wallet API
+  contact: {}
+  version: '1.28.0'
+servers:
+  - url: https://localhost:3010
+    variables:
+      environment:
+        default: dev
+        enum:
+          - dev     # Development server
+          - test    # Test server
+paths:
+  '/{object_type}/{object_uuid}/like':
```

Comment:
User an id rather than name, if use a name, we can not easily change it in the future, and bring the resource type before the id to make it clear:

`/types/{type_id}/things/{object_id}/like` 

The resource name (types, things) can also make the http router definition easier, you resource url: `/{object_type}/{object_uuid}/like` is possible to bring difficulty of setting the route rules.

Reply:
Because we are using id for the type, so we need an endpoint to create/update a type